plutus-ledger/test/Spec.hs

```diff
@@ -229,7 +229,8 @@ ciTxOutRoundTrip :: Property
 ciTxOutRoundTrip = property $ do
   txOuts <- Map.elems . Gen.mockchainUtxo <$> forAll Gen.genMockchain
   forM_ txOuts $ \txOut -> do
-    Hedgehog.assert $ Tx.toTxOut (fromJust $ Tx.fromTxOut txOut) == txOut
+    -- Hedgehog.assert $ Tx.toTxOut (fromJust $ Tx.fromTxOut txOut) == txOut
+    fail "FIXME" -- FIXME
```

Comment:
As mentioned elsewhere `fromTxOut` is broken because `TxOut` do not carry the reference script (but only its hash).